main.py

```diff
@@ -291,13 +292,23 @@ async def main(args):
     description="A Wisp server implementation, written in Python."
   )
 
-  parser.add_argument("--host", default="127.0.0.1", help="The hostname the server will listen on.")
-  parser.add_argument("--port", default=6001, help="The TCP port the server will listen on.")
-  parser.add_argument("--static", help="Where static files are served from.")
-  parser.add_argument("--limits", action="store_true", help="Enable rate limits.")
-  parser.add_argument("--bandwidth", default=1000, help="Bandwidth limit per IP, in kilobytes per second.")
-  parser.add_argument("--connections", default=30, help="Connections limit per IP, in kilobytes per second.")
-  parser.add_argument("--window", default=60, help="Fixed window length for rate limits, in seconds.")
-  args = parser.parse_args()
```

Comment:
There is no reason to remove the CLI arguments.

Reply:
that's how i took the roadmap, remove cli arguments to use a json config